jgrapht-ext/src/test/java/org/jgrapht/ext/DOTExporterTest.java

```diff
@@ -43,7 +43,7 @@
 
     // TODO jvs 23-Dec-2006: externalized diff-based testing framework
 
-    private static final String UNDIRECTED = "graph G {" + NL + "  1 [ label=\"a\" ];" + NL
+    private static final String UNDIRECTED = "strict graph G {" + NL + "  1 [ label=\"a\" ];" + NL
```

Comment:
Could you add one more undirected graph test where strict is not generated? Let's have two tests, one with strict and one without.